go.sum

```diff
@@ -265,6 +265,8 @@ github.com/hashicorp/go-retryablehttp v0.6.7 h1:8/CAEZt/+F7kR7GevNHulKkUjLht3CPm
 github.com/hashicorp/go-retryablehttp v0.6.7/go.mod h1:vAew36LZh98gCBJNLH42IQ1ER/9wtLZZ8meHqQvEYWY=
 github.com/hashicorp/go-rootcerts v1.0.2 h1:jzhAVGtqPKbwpyCPELlgNWhE1znq+qwJtW5Oi2viEzc=
 github.com/hashicorp/go-rootcerts v1.0.2/go.mod h1:pqUvnprVnM5bf7AOirdbb01K4ccR319Vf4pU3K5EGc8=
+github.com/hashicorp/go-secure-stdlib/strutil v0.1.2 h1:kes8mmyCpxJsI7FTwtzRqEy9CdjCtrXrXGuOpxEA7Ts=
+github.com/hashicorp/go-secure-stdlib/strutil v0.1.2/go.mod h1:Gou2R9+il93BqX25LAKCLuM+y9U2T4hlwvT1yprcna4=
```

Comment:
This is used for only 2-3 functions. There's also one strutil function here copied over: https://github.com/hashicorp/consul/blob/main/internal/go-sso/oidcauth/internal/strutil/util.go.

Thoughts on adding this dependency vs copying the functions over?

Reply:
There's `lib/stringslice` in consul that has `stringslice.Contains` which is the same thing as `strutil.StrListContains`

Reply:
That's perfect. I switched lib/stringslice and added the string glob/wildcard function to lib/glob.go (+ unit test)